### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jennahyasin**

### CODE REVIEW

File: cst438_register/src/main/java/com/cst438/controller/CourseController.java
Comment (YukioRivera) [inline cst438_register/src/main/java/com/cst438/controller/CourseController.java:1]: you should add comments to describe what your code is doing

File: cst438_register/src/main/java/com/cst438/service/GradebookServiceMQ.java
Comment (YukioRivera) [inline cst438_register/src/main/java/com/cst438/service/GradebookServiceMQ.java:1]: you should add comments to describe what your code is doing

File: cst438_register/src/main/java/com/cst438/service/GradebookServiceREST.java
Comment (YukioRivera) [inline cst438_register/src/main/java/com/cst438/service/GradebookServiceREST.java:1]: you should add comments to describe what your code is doing

**Review by YukioRivera — COMMENTED** [review]
good code and everything looks like its going to work. No issues with functionality, just add some comments to show what your code is doing

**Comment by jennahyasin** [comment]
Thank you for reviewing my code. I will add some comments to better clarify my code.

**Pull request merged**